membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -337,3 +376,27 @@ object OptionEither {
   }
 
 }
+
+object ListEither {
```

Comment:
not sure if we should remove any of these if they are unused - also perhaps split into a separate file?